crypto/aes/asm/aes-586.pl

```diff
@@ -8,10 +8,10 @@
 
 #
 # ====================================================================
-# Written by Andy Polyakov <appro@openssl.org> for the OpenSSL
+# Written by Andy Polyakov, @dot-asm, initially for use in the OpenSSL
```

Comment:
While all the author sections seem to have been updated, there are still a large number of instances of `appro@openssl.org` after applying this PR - so if that's no longer a valid email address they should be changed too. (There are also some instances of `appro@fy.chalmers.se` - sometimes with `\` before `@` - I don't know if that is still a valid email address)

Note: not sure if we would want to update `CHANGES.md`, where some of these are, but they are also throughout source files, which we should ensure are correct (to my mind, text `CHANGES.md` becomes historical after the release it refers to)

Reply:
updated the remaining stray instances of appro@openssl.org with their github.id.

As for appro@fy.chalmers.se, the only instances I see outside of CHANGES.md (which I agree, is historical after release) is in mips.pl, sha1-ia64.pl, sha1-ppc.pl, and I have no idea if that address is still valid.  Sending a test email to it, gets a reject as I'm not in the allows list of senders, so...unknown.  Going to leave those alone for now.